tests/test_ssds.py

```diff
@@ -21,9 +22,19 @@
 
 S3_SSDS = _S3StagingTest()
 GS_SSDS = _GSStagingTest()
-test_data = TestData()
+
+test_data = TestData(7, ssds.blobstore.AWS_MIN_CHUNK_SIZE + 1)
```

Comment:
Why the `+ 1`?

Reply:
This trips it one byte past the multipart threshold.